Havit.Blazor.Components.Web.Bootstrap/Navigation/HxSidebarItem.razor

```diff
@@ -12,13 +12,21 @@
                 <div data-bs-toggle="collapse" data-bs-target="#@ParentSidebar.NavContentElementId" class="hx-sidebar-item-nav-content stretched-link">
 
                     <div class="hx-sidebar-nav-content-inner">
-                        @if (Icon != null)
+                        @if (ContentTemplate is null)
                         {
-                            <HxIcon Icon="Icon" />
+                            if (Icon != null)
+                            {
+                                <HxIcon Icon="Icon" />
+                            }
+                            
+                            <span class="@CssClassHelper.Combine("text", Icon == null ? null : "ms-3")">
+                                @Text
+                            </span>
+                        }
+                        else
+                        {
+                            @ContentTemplate
                         }
```

Comment:
We should not replace the current item with the `ContentTemplate`, but add it to the other options (similar patter is used in other components where it makes sense).
I would go for order (chevron icon just for expandable items)
```
  @if ... <HxIcon />
  @ContentTemplate
  <span ...>@Text</span>
  <HxIcon Icon="@BootstrapIcon.ChevronRight" CssClass="expand-icon" />
```